packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -387,12 +409,18 @@ function UserMenu({
         ) : (
           <MenuItem
             {...menuItemContainer.prop}
-            href={hosts.cloud}
+            href={isGovernment ? 'https://cloud.mongodbgov.com' : hosts.cloud}
```

Comment:
I wonder if we just make this `hosts.cloud` so then we can customize it to be `cloud.mongodbgov.com` as we desire.

And then when we initially construct hosts in MongoNav we check if environment is government and set it to `mongodbgov.com`

ditto above

Reply:
Ah, because the domain would be different everywhere.. that makes sense.

Reply:
Updated and added tests